File: php/Mahasiswa.php
<?php

class Mahasiswa
{
    private $nim;
    private $nama;
    private $jenis_kelamin;
    private $fakultas;
    private $prodi;

    function __construct($nim = null, $nama = null, $jenis_kelamin = null, $fakultas = null, $prodi = null)
    {
        $this->nim = $nim;
        $this->nama = $nama;
        $this->jenis_kelamin = $jenis_kelamin;
        $this->fakultas = $fakultas;
        $this->prodi = $prodi;
    }

    function setNim($nim)
    {
        $this->nim = $nim;
    }

    function getNim()
    {
        return $this->nim;
    }

    function setNama($nama)
    {
        $this->nama = $nama;
    }

    function getNama()
    {
        return $this->nama;
    }

    function setJk($jenis_kelamin)
    {
        $this->jenis_kelamin = $jenis_kelamin;
    }

    function getJk()
    {
        return $this->jenis_kelamin;
    }

    function setFk($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    function getFk()
    {
        return $this->fakultas;
    }

    function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }

    function getProdi()
    {
        return $this->prodi;
    }
}

?>

File: python/mahasiswa.py
class mahasiswa:
    __nim = ""
    __nama = ""
    __jk = ""
    __fk = ""
    __prodi = ""

    def __init__(self, nim="", nama="", jk="", fk="", prodi=""):
        self.__nim = nim
        self.__nama = nama
        self.__jk = jk
        self.__fk = fk
        self.__prodi = prodi

    def setNim(self, nim):
        self.__nim = nim

    def getNim(self):
        return self.__nim

    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setJk(self, jk):
        self.__jk = jk

    def getJk(self):
        return self.__jk

    def setFk(self, fk):
        self.__fk = fk

    def getFk(self):
        return self.__fk

    def setProdi(self, prodi):
        self.__prodi = prodi

    def getProdi(self):
        return self.__prodi
